ci/after_success.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Echo all commands before executing them
+set -o xtrace
+# Forbid any unset variables
+set -o nounset
+# Exit on any error
+set -o errexit
+
+# We only need to run the coverage suite once
+COVERAGE_RUN=false
+KCOV_BIN="./target/kcov-master/build/src/kcov"
+
+run_kcov() {
+    # Run kcov on all the test suites
+    if [[ $COVERAGE_RUN != "true" ]]; then
+        cargo coverage --all --tests --benches --no-default-features
+        mv target/kcov target/kcov-no-default-features
+        cargo coverage --all --tests --benches --features "$FEATURES"
+        mv target/kcov target/kcov-features
```

Comment:
The commands above don't run all the tests. (They miss the ones in `serialization-tests`, `blas-tests`, and `numeric-tests`.)

Reply:
That's odd; the `--all` flag should run all tests from all workspaces.  Could this be left as-is and once this pull request in merged, I'll work on #637?

Reply:
numeric-tests is explicitly not in the workspace, so that it can use its own profile/compilation settings. It can be added if that can be achieved another way

Reply:
I don't see really see anything special about `numeric-tests` which can't be included as part of #637.  If it is because of a specific combination of features, then we can easily add them by adding to the Travis-CI test matrix.  If there's something more subtle, perhaps this can be addressed in #637?